quint/src/cliCommands.ts

```diff
@@ -322,7 +322,8 @@ export async function runRepl(_argv: any) {
 export async function runTests(prev: TypecheckedStage): Promise<CLIProcedure<TestedStage>> {
   const testing = { ...prev, stage: 'testing' as stage }
   const verbosityLevel = deriveVerbosity(prev.args)
-  const mainName = guessMainModule(prev)
+  const guessedMainModule = guessMainModule(prev)
+  const mainName = prev.args.main || guessedMainModule
```

Comment:
This change is not necessary, as `guessMainModule` already returns `prev.args.main` if provided.
```suggestion
  const mainName = guessMainModule(prev)
```